src/DataMigration/DataMigration.Autorest/custom/Cmdlets/New-AzDataMigrationLoginsMigration.ps1

```diff
@@ -78,77 +78,179 @@ function New-AzDataMigrationLoginsMigration
             #Defining Default Output Path
             $DefaultOutputFolder = Get-DefaultLoginMigrationsOutputFolder
 
-            #Defining Base and Exe paths
-            $BaseFolder = Join-Path -Path $DefaultOutputFolder -ChildPath Downloads;
-            $ExePath = Join-Path -Path $BaseFolder -ChildPath Logins.Console.csproj\Logins.Console.exe;
+            #Defining Downloads folder
+            $DownloadsFolder = Join-Path -Path $DefaultOutputFolder -ChildPath Downloads;
 
-            #Checking if BaseFolder Path is valid or not
-            if(-Not (Test-Path $BaseFolder))
+            #Checking if DownloadsFolder Path is valid or not
+            if(-Not (Test-Path $DownloadsFolder))
             {
-                $null = New-Item -Path $BaseFolder -ItemType "directory"
+                $null = New-Item -Path $DownloadsFolder -ItemType "directory"
+            }
+            else
+            {
+                #Remove old Login console app files
+                #Old console app download address
+                $ZipDestination = Join-Path -Path $DownloadsFolder -ChildPath "LoginsMigration.zip";
+
+                # Remove old zip file
+                if(Test-Path $ZipDestination)
+                {
+                    Remove-Item -Path $ZipDestination;
```

Comment:
We are removing files three times. We can probably create a function to test path and remove item to avoid repetative code

Reply:
That should be fine for now. We will be removing this part of the code in the next iteration. This is added since the console app is renamed from Logins.Console to Microsoft.SqlServer.Migration.Logins.ConsoleApp.